internal/display/display_test.go

```diff
@@ -47,7 +48,8 @@ func TestNewDisplayWriter(t *testing.T) {
 				t.Fatal(err)
 			}
 
-			if gotOutput := defaultOutput.String(); gotOutput != tt.wantOutput {
+			// Trim the output to remove any extra whitespace or newlines
+			if gotOutput := strings.TrimSpace(defaultOutput.String()); gotOutput != tt.wantOutput {
```

Comment:
let's trim space on GOOS == windows.

Reply:
Are you asking to execute the `trimSpace` function only if the OS is Windows? If so, I've made the necessary changes; please check them.